Channel1:
   ForceConversion: 1168103
Channel2:
   ForceConversion: 1168103
Channel3:
   ForceConversion: 1168103
Channel4:
   ForceConversion: 1168103
   